src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
```

Comment:
- [ ] Kindly pass the payload here in the action creator, So the reducer can get the payload at the case when the `addBook` action is dispatched.
[Hint] It will be something like that:
```javascript
export const addBook = (payload) => ({
  type: ADD_BOOK,
  payload,
});
```

Reply:
done